Page: add Go-to-top button

grzegorz-czajkowski opened the pull request:
[Issue] https://github.com/Samsung/TAU/issues/628
[Problem] No Go-to-top button in content example
[Solution] Add Go-to-top button to Page widget according to
   the guideline and implement it for Content.
[Remarks] On lists, the button does not work since it will be
   handled in https://github.com/Samsung/TAU/issues/627.
    
 Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

grzegorz-czajkowski committed:
Page: add Go-to-top button

[Issue] https://github.com/Samsung/TAU/issues/628
[Problem] No Go-to-top button in content example
[Solution] Add Go-to-top button to Page widget according to
	the guideline and implement it for Content.
[Remarks] On lists, the button does not work since it will be
	handled in https://github.com/Samsung/TAU/issues/627.

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

TomaszLukawskiSam commented on src/css/profile/wearable/changeable/common/button.less:
```diff
@@ -355,3 +355,10 @@ button.ui-btn, input.ui-btn {
+.ui-button-go-to-top {
+	width: 52 * @unit_base;
+	height: 52 * @unit_base;
+	background-image: url("./images/GoToTopButton/go_to_top_dark.png");
```
I think we shouldn't create additional directory like 'GoToTopButton' to store images. This makes additional issues when images will be updated.

grzegorz-czajkowski replied:
We are having images in dirs https://github.com/Samsung/TAU/tree/tau_1.0/src/css/profile/wearable/changeable/theme-changeable/images

TomaszLukawskiSam replied:
I thought about not creating additional subdirectories `GoToTopButton`

grzegorz-czajkowski commented:
As we discussed, I am changing the approach to add the button to the page (instead of scrollview). This will allow us to handle click on the button for arclistview.

grzegorz-czajkowski committed:
Page: add enableGoToTopButton option

[Issue] https://github.com/Samsung/TAU/issues/628
[Problem] GoToTopButton is added to every Page widget whereas
	it's expected to be available on content and lists.
[Solution] add enableGoToTopButton option to Page and enable it
	on content. On lists it is not implemented yet.

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

grzegorz-czajkowski commented:
Added commit to enable go top button by Page option.

TomaszLukawskiSam reviewed (approved):
Looks good to me.
